packages/design-system/src/components/mediaInput/index.js

```diff
@@ -221,6 +236,7 @@ export const MediaInput = forwardRef(function Media(
               internalRef.current = node;
             }}
             id={buttonId}
+            $mediaVariant={variant}
             variant={BUTTON_VARIANTS.SQUARE}
```

Comment:
Don't love this, but there's two variants at play here:
1. the variant of the `MediaInput` component (Showing or not showing the story preview)
2. the variant of the button

Reply:
`$mediaVariant` could be changed to `showImage={variant === MEDIA_VARIANT.NONE}`. This would be simpler and more readable imo

Reply:
i like that idea more, agree that it's more readable.